src/e2e-test/features/dataplex/source/RunTimeDataplex.feature

```diff
@@ -0,0 +1,101 @@
+    And Click on the Validate button
+    And Close the Plugin Properties page
+    And Navigate to the properties page of plugin: "Dataplex2"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexAssetID" with value: "dataplex.assetBQ"
+    And Enter input plugin property: "dataplexTableName" with value: "dataplex.newTable"
+    And Click on the Validate button
+    And Close the Plugin Properties page
+    And Save the pipeline
+    And Preview and run the pipeline
+    And Verify the preview of pipeline is "success"
+
+  @BATCH-TS-DATAPLEX-RunTime-02
+  Scenario: To verify the pipeline is deployed and verify whether pipeline is successful when asset type is selected
+  as bigquery dataset in sink
+    When Open Datafusion Project to configure pipeline
+    Then Select plugin: "Dataplex" from the plugins list as: "Source"
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Connect plugins: "Dataplex" and "Dataplex2" to establish connection
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexEntityID" with value: "dataplex.entity"
+    And Click on the Get Schema button
+    And Click on the Validate button
+    And Close the Plugin Properties page
+    And Navigate to the properties page of plugin: "Dataplex2"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexAssetID" with value: "dataplex.assetBQ"
+    And Enter input plugin property: "dataplexTableName" with value: "dataplex.newTable"
+    And Click on the Validate button
+    And Close the Plugin Properties page
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Wait till pipeline is in running state
+    And Open and capture logs
+    And Verify the pipeline status is "Succeeded"
+
+  @BATCH-TS-DATAPLEX-RunTime-03
+  Scenario Outline: To verify the pipelines are deployed and is successfull when asset type is storage bucket and
```

Comment:
-> To verify the pipeline is successfully deployed and run for different format when asset type is storage bucket